istio/tests/test_e2e.py

```diff
@@ -14,6 +14,21 @@
     'istio.pilot.mcp_sink.recv_failures_total',
     'istio.galley.validation.passed',
     'istio.pilot.rds_expired_nonce',
+    'istio.galley.validation.config_update_error.count',
```

Comment:
Are these metrics deprecated in the newest version of Istio? We may want to also update the `metadata.csv` for these metrics if that's the case.

Reply:
Not really sure. Seems like these metrics are still exported:
https://istio.io/latest/docs/reference/commands/istioctl/#metrics

Not sure if there's an action/trigger that needs to happen for these metrics to be generated